CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [3.3.2] - 2023-01-24
+
+### Added
+- Added the `partytown` library to enable shopkeepers to load third party scripts off the main thread.
+
```

Comment:
The new changes should be in the `Unreleased` section. Our bot updates it with the new version.

```suggestion
### Added
- `partytown` option to enable shopkeepers to load third-party scripts off the main thread.

```

Reply:
> The new changes should be in the `Unreleased` section. Our bot updates it with the new version.

Done!